ci/build-canton-3x.sh

```diff
@@ -15,9 +15,11 @@ else
   repo_url="https://$GITHUB_TOKEN@github.com/DACH-NY/canton"
 fi
 
-git clone --depth 1 --branch main $repo_url $tmp
-head=$(git -C $tmp rev-parse HEAD)
-echo "cloned at revision $head"
+commitish=${1:-$(cat canton-3x/canton-3x-sha)}
```

Comment:
This is a relative path; probably worth adding a
```
DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```
at the top and updating this to
```suggestion
commitish=${1:-$(cat "$DIR"/../canton-3x/canton-3x-sha)}
```

Reply:
Done.